odoo/addons/test_new_api/tests/test_properties.py

```diff
@@ -1849,3 +1849,11 @@ def test_properties_field_search_orderby_injection(self):
             for order in orders:
                 with self.assertRaises(UserError), self.assertQueryCount(0):
                     self.env['test_new_api.message'].search(domain=[], order=order)
+
+    @mute_logger('odoo.fields')
+    def test_properties_field_search(self):
```

Comment:
Edit: it's even worst than I tough, it will search on `properties->"None"`
```sql
( ("test_new_api_message"."attributes" ->> 'None') = (%s))
```

Reply:
Hello @std-odoo ,
I didn't got this point of yours. Otherwise I have added the test case which is working with the fix, can you please check it.
Thanks :slightly_smiling_face: